registry/server/util/pluginManager.ts

```diff
@@ -0,0 +1,17 @@
+import { PluginManager } from 'ilc-plugins-sdk';
+import manifest from '../../server.plugins.manifest.json';
+
+let pluginManagerInstance: PluginManager;
+
+export function loadPlugins() {
+    if (!pluginManagerInstance) {
+        pluginManagerInstance = new PluginManager(...manifest.manifest.plugins);
```

Comment:
is there any potential issue to initialize it directly in module? 
i.e. `const pluginManagerInstance = new PluginManager(...)`

Reply:
it will be triggered in the import ESM module phase, so it makes it harder to control flow.